### DISCUSSION TITLE
Moving from HACS back into Integrations

### DISCUSSION THREAD

**Discussion by HandHsince1920** [dbuezas/eq3btsmart#117] [General]
Hello,

I have recently seen that eq3bt is back in the normal HA integrations section. When It tried it I was surprised though that functionally it is not on a par with the actual HACS integration.
Is there a plan to bring the extended functionality into the normal HA integration?

**Comment by dbuezas** [comment]
The version in HA core is a fork of this repo by @EuleMitKeule, you'll have to ask him :)

**Comment by EuleMitKeule** [comment]
I had to patch it out because HA maintainers only allowed submitting a single platform/entity in a single PR. I am currently learning for exams and hadn't had time to open more PRs with the remaining functionality.

**Reply by HandHsince1920** [reply]
Fingers crossed for you exams, no pressure. I just wondered.

**Comment by renarena** [comment]
is there still any developement? Like:
    add external tmp sensors
    read/write heating plans

**Reply by EuleMitKeule** [reply]
If this is about the core integration, this is not really the right place to ask such questions, but I will answer:

> is there still any developement?

Yes, in the last few days I fixed a few bugs, and am currently finalizing a larger update that generally improves the integration and introduces the ability to disable automatic status updates which should drastically reduce the battery consumption.

> add external tmp sensors

This is not wanted by the HA core team. You can however create your own "virtual" climate entity that combines the eq3 thermostat with external temperature sensors with my custom integration [template-climate](https://github.com/eulemitkeule/template-climate). That is how I am doing it.

> read/write heating plans

Schedules are already supported by the underlying [eq3btsmart library](https://github.com/eulemitkeule/eq3btsmart). I am planning to integrate them into the core integration soon.
